lib/teiserver/mix_tasks/fake_data.ex

```diff
@@ -458,4 +459,24 @@ defmodule Mix.Tasks.Teiserver.Fakedata do
       :rand.uniform() < chance
     end)
   end
+
+  # Adds a party id or nil
+  defp get_party_id(num_players) do
+    case is_in_party?() do
+      true ->
+        num_parties = trunc(num_players / 4)
+        # party id is a string
+        "#{Enum.random(1..num_parties)}"
+
+      false ->
+        nil
+    end
+  end
+
+  defp is_in_party? do
+    chance_in_party = 40
+    # Number from 0 to 100
+    random = Enum.random(0..100)
+    random <= chance_in_party
```

Comment:
Just for my own curiosity, is there a quicker way to do this? How's elixir on random numbers?

Reply:
https://stackoverflow.com/a/42295898
I just copied this

Reply:
For this usecase you can also use `:rand.uniform/1` from [erlang stdlib](https://www.erlang.org/doc/apps/stdlib/rand.html#uniform/1) like for the entire function
```
:rand.uniform(100) <= 40
```